modules/mesos-cluster/main.tf

```diff
@@ -229,3 +230,21 @@ resource "aws_route53_record" "paas-mozilla-org-dns-zone" {
   ttl = 300
   records = ["${aws_elb.mozilla-org-elb.dns_name}"]
 }
+
+resource "aws_cloudwatch_metric_alarm" "cloudwatch-health-host-count-elb-alarm" {
+  alarm_name = "cloudwatch-health-count-${var.environment}"
+  comparison_operator = "LessThanThreshold"
+  metric_name = "HealthyHostCount"
+  threshold = "3"
+  statistic = "Average"
+  period = "120"
+  alarm_description = "Monitor number of healthy instances."
+  ok_actions = []
+  insufficient_data_actions = []
+  namespace = "AWS/ELB"
+  dimensions = {
+    "LoadBalancerName"="mesos-${var.environment}-elb"
+  }
+  actions_enabled = true
+  alarm_actions = ["arn:aws:sns:${var.aws_region}:${var.aws_account_id}:aws_sns_topic/sns-cloudwatch-partinfra"]
```

Comment:
After we have set the variable we can just use `${var.sns_topic_arn}` here. That way we would not need to change anything if we deploy this in a different region.